.eleventy.js

```diff
@@ -1,3 +1,15 @@
+/*
+Copyright the Infusion copyright holders.
```

Comment:
Sorry, I wasn't clear about this. It wouldn't be "Infusion" here. I tend to just name them after the repo because each codebase has different copyright holders. It's more a way of grouping the individual copyright holders defined in the AUTHORS.md file.